examples/http/tracer.js

```diff
@@ -10,6 +11,20 @@ const EXPORTER = process.env.EXPORTER || '';
 
 module.exports = (serviceName) => {
   const provider = new NodeTracerProvider();
+  registerInstrumentations({
+    tracerProvider: provider,
+    // // when boostraping with lerna for testing purposes
+    // instrumentations: [
+    //   {
+    //     plugins: {
+    //       http: {
+    //         enabled: true,
+    //         path: `${__dirname}/../../packages/opentelemetry-plugin-http/build/src`
```

Comment:
There exists already an instrumentation for http/https

Reply:
ok but this if for testing old plugins when using lerna as mentioned above it.